setup/create_venv

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -ex
+
+venv_location=/home/ofuhrer/install/
+
+# get directory where this script is located in
+dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+# go to target directory
+cd ${venv_location}
+
+# make sure we are using the right modules
+module purge
+source ${dir}/module_env
+
+# create virtual environment for workshop
+python -m venv venv
+source venv/bin/activate
+
+# install some standard packages we need
+pip install --upgrade pip setuptools wheel
+pip install numpy matplotlib ipykernel ipyparallel mpi4py
+
+# install GT4Py
+if [ ! -d gt4py ] ; then
+  #git clone git@github.com:VulcanClimateModeling/gt4py.git
+  git clone git@github.com:GridTools/gt4py.git
```

Comment:
hopefully we can switch this back before the sessions

Reply:
Agreed.